uniflow/flow/config.py

```diff
@@ -181,9 +181,24 @@ class RaterClassificationConfig(RaterConfig):
         default_factory=lambda: {"Yes": 1.0, "No": 0.0}
     )
     guided_prompt_template: GuidedPrompt = GuidedPrompt(
-        instruction="""Rate the answer based on the question and the context.
-        Follow the format of the examples below to include context, question, answer, and label in the response.
-        The response should not include examples in the prompt.""",
+        instruction="""
+        Task: Answer Evaluation
+        Objective:
+        You are required to evaluate whether a given answer is appropriate in relation to a specific context and question. 
+        Input:
+        1. Context: This is a brief text, usually a couple of sentences or a paragraph, providing key information or facts.
+        2. Question: This is a query related to the information given in the context. It is designed to test knowledge that can be inferred or directly obtained from the context.
+        3. Answer: This is a response to the question provided.
+        Evaluation Criteria:
+        Based on these, you need to judge if the answer is correct or incorrect in relation to the context and the question. Use the following labels to categorize your judgment:
+        1. Yes: The answer is good.
+        2. No: The answer is not bad.
+        Response Format:
+        Your response should include:
+        1. label: Your judgment (one of the two labels mentioned above).
+        2. explanatoin: A clear and concise thought for your judgment, detailing why you think the answer is good or bad.
+        Note: Only use the example below as a few shot demonstrate but not including them in the final response.
+        """,
```

Comment:
Any reason using such a long prompt? It will really expensive for the users. Can you try to shorten it into half?

Reply:
Because when using GPT-3.5, it seems you have to make a excessive clear (even kind of verbose) so model can response well (). If default is GPT4, short prompt is OK. I can try to shorten it.